include/sunadaptcontroller/sunadaptcontroller_mrihtol.h

```diff
@@ -0,0 +1,88 @@
+/* -----------------------------------------------------------------
+ * Programmer(s): Daniel R. Reynolds @ SMU
+ * -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2024, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------
+ * This is the header file for the SUNAdaptController_MRIHTol module.
+ * -----------------------------------------------------------------*/
+
+#ifndef _SUNADAPTCONTROLLER_MRIHTOL_H
+#define _SUNADAPTCONTROLLER_MRIHTOL_H
+
+#include <stdio.h>
+#include <sundials/sundials_adaptcontroller.h>
+
+#ifdef __cplusplus /* wrapper to enable C++ usage */
+extern "C" {
+#endif
+
+/* --------------------------------------------
+ * MRI H+tolerance implementation of SUNAdaptController
+ * -------------------------------------------- */
+
+struct _SUNAdaptControllerContent_MRIHTol
```

Comment:
Switch to something without a underscore to start (reserved)

Reply:
This matches our other class implementations, and I believe that the underscore is still required for our Swig setup to run properly.  I recommend that we just add this request to the list of things-to-change-once-swig-changes.